doc/github_actions.rst

```diff
@@ -0,0 +1,35 @@
+Integration with GitHub Actions
+===============================
+
+`Universum` requires no special integration with `GitHub Actions <https://docs.github.com/en/actions>`_. It is usually
+launched as one long step in a single build stage.
+
+.. warning::
+
+    GitHub Actions doesn't support `report_artifacts` key in configuration.
+
+Command line
+------------
+
+To use `universum` to check-out current repository (the repository that contains the GitHub Action workflow) is
+necessary to use `GitHub Environment variables
+<https://docs.github.com/en/actions/learn-github-actions/environment-variables>`_:
+``python -m universum --vcs-type=git --git-repo $GITHUB_SERVER_URL/$GITHUB_REPOSITORY --git-refspec $GITHUB_REF_NAME``
+
+`Universum` also supports reports to review systems from GitHub Actions. Link to build are created using environment variables
+and do not require any additional command line parameters.
+
+Logs
+----
+
+Github Actions currently supports single-level grouping of log lines
+(`without nesting <https://github.com/actions/runner/issues/802>`_). Accordingly, logs is printed with the following
```

Comment:
Because of that, Universum logs are printed using the following restrictions: an already opened group is closed if any other (including nested) group is opened.

Reply:
Yes, you are right. But this only works for nested groups, because only when nested groups are opened, the  already opened group is not closed yet.